server/repository/src/migrations/v1_01_01/mod.rs

```diff
@@ -13,6 +13,8 @@ impl Migration for V1_01_01 {
     }
 
     fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        use crate::migrations::sql;
```

Comment:
will also fix in develop. it's arrived as a merge error which prevents building the server.